src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/integTests/tests/jvmsizing/resources/rca_master_high_threshold.conf

```diff
@@ -0,0 +1,93 @@
+{
+  "analysis-graph-implementor":
+    "com.amazon.opendistro.elasticsearch.performanceanalyzer.rca.store.AnalysisGraphTest",
+  // it can be file:// or s3://
+  "rca-store-location": "s3://sifi-store/rcas/",
+
+  //it can be file:// or s3://
+  "threshold-store-location": "s3://sifi-store/thresholds/",
+
+  "new-rca-check-minutes": 60,
+
+  "new-threshold-check-minutes": 30,
+
+  // The size of the task queue for all networking operations.
+  // Small size queues may result in dropping of flow units, while large size queues can lead to a
+  // bigger backlog of tasks resulting in delays in sending and receiving.
+  "network-queue-length": 200,
+
+  // The size of the per-vertex buffer for flow units received from remote nodes.
+  // Small buffer sizes may result in dropping of flow units, while large buffer size can lead to
+  // high memory consumptions depending on how the analysis graph is configured.
+  "max-flow-units-per-vertex-buffer": 200,
+
+  "tags": {
+    "locus": "master-node",
+    "disk": "ssd",
+    "region": "use1",
+    "instance-type": "i3.8xl",
+    "domain": "rca-test-cluster"
+  },
+
+  "remote-peers": ["ip1", "ip2", "ip3"],
+
+  // Tells the runtime where the RCAs will be stored.
+  "datastore": {
+    // accepted types are sqlite, in-memory.
+    "type": "sqlite",
+    "location-dir": "/tmp",
+    "filename": "rca.sqlite",
+    "storage-file-retention-count": 5,
+    // How often the sqlite file be repeated in seconds. This file contains RCAs and therefore rotating it too frequently
+    // might not be as fruitful as there might not be any data.
+    "rotation-period-seconds": 21600
+  },
+
+  // Add config settings for different RCAs
+  "rca-config-settings": {
+    // old gen rca
+    "high-heap-usage-old-gen-rca": {
+      "top-k" : 3
+    },
+    "high-old-gen-occupancy-config": {
+      "eval-interval-in-s": 5,
+      "heap-utilization-threshold": 60
+    },
+    "HeapSizeIncreasePolicy": {
```

Comment:
where are we actively using this `HeapSizeIncreasePolicy` config? or maybe `heap-size-increase-policy` config is good enough?

Reply:
HeapSizeIncreasePolicy config is used by the decider.